core/templates/pages/community-dashboard-page/services/question-suggestion.service.ts

```diff
@@ -28,8 +28,8 @@ angular.module('oppia').factory('QuestionSuggestionService', [
           description: 'Add new question',
           target_id: associatedSkill.getId(),
           target_version_at_submission: associatedSkill.getVersion(),
-          assigned_reviewer_id: '',
-          final_reviewer_id: '',
+          assigned_reviewer_id: null,
+          final_reviewer_id: null,
```

Comment:
I would suggest removing these two fields from the request for two reasons:
1. Backend should set it manually instead of depending on the client.
2. We don't expect the client to know their or someone else user_id.

Reply:
Ditto for the translation suggestion request. [Make sure to test exploration edit content suggestion flow if we are going to follow this pattern.]

Reply:
Done.